pkg/logstructured/logstructured.go

```diff
@@ -5,8 +5,14 @@ import (
 	"sync"
 	"time"
 
-	"github.com/k3s-io/kine/pkg/server"
 	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/util/workqueue"
+
+	"github.com/k3s-io/kine/pkg/server"
+)
+
+const (
+	retryInterval = 5 * time.Second
```

Comment:
This could probably be shorter - maybe start out at 250ms? Just kind of guessing.